.goreleaser.yml

```diff
@@ -36,8 +36,11 @@ checksum:
 signs:
   - artifacts: checksum
     args:
+      # if you are using this is a GitHub action or some other automated pipeline, you 
+      # may want to pass the batch flag to indicate its not interactive.
+      # - "--batch" 
       - "--local-user"
-      - "51852D87348FFC4C" # Replace this with your GPG signing key ID
+      - "{{ .Env.GPG_FINGERPRINT }}" # set this environment variable for your signing key
```

Comment:
I see that this is needed for the GitHub Action, but this might trip someone up who is not using that. IF the Action works well though, maybe we just want to make that the suggested publishing mechanism. We could also extract the local-publishing example to docs or elsewhere and keep this just for GitHub Actions + GoReleaser.

Reply:
Brought this up internally, but I think we should just bless goreleaser + GitHub actions as the preferred publishing method.

Reply:
Sounds good! Let's do that.